fetch.bs

```diff
@@ -1084,10 +1085,10 @@ Unless stated otherwise, it is "<code>basic</code>".
 <hr>
 
 <p>A <dfn export>subresource request</dfn> is a <a for=/>request</a>
-whose <a for=request>destination</a> is "<code>font</code>",
-"<code>image</code>", "<code>manifest</code>", "<code>media</code>",
-"<code>script</code>", "<code>style</code>", "<code>xslt</code>", or the empty
-string.
+whose <a for=request>destination</a> is "<code>fetch</code>",
+"<code>font</code>", "<code>image</code>", "<code>manifest</code>",
+"<code>media</code>", "<code>script</code>", "<code>style</code>",
+"<code>xslt</code>", or the empty string.
```

Comment:
If I properly understand your suggestion here, you'd prefer to keep the destination as is (i.e. not add a "fetch" destination), create the concept of "potential destination", have preload use that, and just translate is to destination (by having preload call "normalize potential destination" or somesuch) before preload triggers a fetch. Is that correct?